mne/io/ctf/ctf.py

```diff
@@ -129,6 +129,7 @@ def __init__(self, directory, system_clock='truncate', preload=False,
             if sample_info['n_samp'] == 0:
                 break
             if len(fnames) == 0:
+                # XXX: Buffer size different to c version
                 info['buffer_size_sec'] = \
                     sample_info['block_size'] / info['sfreq']
```

Comment:
Thanks for looking into it. It will be good for you to take a look instead of me so that more than one person has a handle on what the code is doing.